searchState.loadedDescShard("spinoso_exception", 0, "Built in Ruby exception types.\nRuby <code>ArgumentError</code> error type.\nRuby <code>EOFError</code> error type.\nRuby <code>EncodingError</code> error type.\nRuby <code>Exception</code> error type.\nRuby <code>fatal</code> error type.\nRuby <code>FiberError</code> error type.\nRuby <code>FloatDomainError</code> error type.\nRuby <code>FrozenError</code> error type.\nRuby <code>IOError</code> error type.\nRuby <code>IndexError</code> error type.\nRuby <code>Interrupt</code> error type.\nRuby <code>KeyError</code> error type.\nRuby <code>LoadError</code> error type.\nRuby <code>LocalJumpError</code> error type.\nRuby <code>NameError</code> error type.\nRuby <code>NoMemoryError</code> error type.\nRuby <code>NoMethodError</code> error type.\nRuby <code>NotImplementedError</code> error type.\nRuby <code>RangeError</code> error type.\nRuby <code>RegexpError</code> error type.\nPolymorphic exception type that corresponds to Ruby’s …\nRuby <code>RuntimeError</code> error type.\nRuby <code>ScriptError</code> error type.\nRuby <code>SecurityError</code> error type.\nRuby <code>SignalException</code> error type.\nRuby <code>StandardError</code> error type.\nRuby <code>StopIteration</code> error type.\nRuby <code>SyntaxError</code> error type.\nRuby <code>SystemCallError</code> error type.\nRuby <code>SystemExit</code> error type.\nRuby <code>SystemStackError</code> error type.\nRuby <code>ThreadError</code> error type.\nRuby <code>TypeError</code> error type.\nRuby <code>UncaughtThrowError</code> error type.\nRuby <code>ZeroDivisionError</code> error type.\nRuby exception class implementations.\nThe exception’s message.\nThe exception’s class name.\nRuby <code>ArgumentError</code> error type.\nRuby <code>EOFError</code> error type.\nRuby <code>EncodingError</code> error type.\nRuby <code>Exception</code> error type.\nRuby <code>fatal</code> error type.\nRuby <code>FiberError</code> error type.\nRuby <code>FloatDomainError</code> error type.\nRuby <code>FrozenError</code> error type.\nRuby <code>IOError</code> error type.\nRuby <code>IndexError</code> error type.\nRuby <code>Interrupt</code> error type.\nRuby <code>KeyError</code> error type.\nRuby <code>LoadError</code> error type.\nRuby <code>LocalJumpError</code> error type.\nRuby <code>NameError</code> error type.\nRuby <code>NoMemoryError</code> error type.\nRuby <code>NoMethodError</code> error type.\nRuby <code>NotImplementedError</code> error type.\nRuby <code>RangeError</code> error type.\nRuby <code>RegexpError</code> error type.\nRuby <code>RuntimeError</code> error type.\nRuby <code>ScriptError</code> error type.\nRuby <code>SecurityError</code> error type.\nRuby <code>SignalException</code> error type.\nRuby <code>StandardError</code> error type.\nRuby <code>StopIteration</code> error type.\nRuby <code>SyntaxError</code> error type.\nRuby <code>SystemCallError</code> error type.\nRuby <code>SystemExit</code> error type.\nRuby <code>SystemStackError</code> error type.\nRuby <code>ThreadError</code> error type.\nRuby <code>TypeError</code> error type.\nRuby <code>UncaughtThrowError</code> error type.\nRuby <code>ZeroDivisionError</code> error type.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nReturns the argument unchanged.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nCalls <code>U::from(self)</code>.\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn the message this Ruby exception was constructed …\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nReturn this Ruby exception’s class name.\nConstruct a new, default <code>ArgumentError</code> Ruby exception.\nConstruct a new, default <code>EncodingError</code> Ruby exception.\nConstruct a new, default <code>EOFError</code> Ruby exception.\nConstruct a new, default <code>Exception</code> Ruby exception.\nConstruct a new, default <code>fatal</code> Ruby exception.\nConstruct a new, default <code>FiberError</code> Ruby exception.\nConstruct a new, default <code>FloatDomainError</code> Ruby exception.\nConstruct a new, default <code>FrozenError</code> Ruby exception.\nConstruct a new, default <code>IndexError</code> Ruby exception.\nConstruct a new, default <code>Interrupt</code> Ruby exception.\nConstruct a new, default <code>IOError</code> Ruby exception.\nConstruct a new, default <code>KeyError</code> Ruby exception.\nConstruct a new, default <code>LoadError</code> Ruby exception.\nConstruct a new, default <code>LocalJumpError</code> Ruby exception.\nConstruct a new, default <code>NameError</code> Ruby exception.\nConstruct a new, default <code>NoMemoryError</code> Ruby exception.\nConstruct a new, default <code>NoMethodError</code> Ruby exception.\nConstruct a new, default <code>NotImplementedError</code> Ruby …\nConstruct a new, default <code>RangeError</code> Ruby exception.\nConstruct a new, default <code>RegexpError</code> Ruby exception.\nConstruct a new, default <code>RuntimeError</code> Ruby exception.\nConstruct a new, default <code>ScriptError</code> Ruby exception.\nConstruct a new, default <code>SecurityError</code> Ruby exception.\nConstruct a new, default <code>SignalException</code> Ruby exception.\nConstruct a new, default <code>StandardError</code> Ruby exception.\nConstruct a new, default <code>StopIteration</code> Ruby exception.\nConstruct a new, default <code>SyntaxError</code> Ruby exception.\nConstruct a new, default <code>SystemCallError</code> Ruby exception.\nConstruct a new, default <code>SystemExit</code> Ruby exception.\nConstruct a new, default <code>SystemStackError</code> Ruby exception.\nConstruct a new, default <code>ThreadError</code> Ruby exception.\nConstruct a new, default <code>TypeError</code> Ruby exception.\nConstruct a new, default <code>UncaughtThrowError</code> Ruby exception.\nConstruct a new, default <code>ZeroDivisionError</code> Ruby exception.\nConstruct a new, <code>ArgumentError</code> Ruby exception with the …\nConstruct a new, <code>EncodingError</code> Ruby exception with the …\nConstruct a new, <code>EOFError</code> Ruby exception with the given …\nConstruct a new, <code>Exception</code> Ruby exception with the given …\nConstruct a new, <code>fatal</code> Ruby exception with the given …\nConstruct a new, <code>FiberError</code> Ruby exception with the given …\nConstruct a new, <code>FloatDomainError</code> Ruby exception with the …\nConstruct a new, <code>FrozenError</code> Ruby exception with the given …\nConstruct a new, <code>IndexError</code> Ruby exception with the given …\nConstruct a new, <code>Interrupt</code> Ruby exception with the given …\nConstruct a new, <code>IOError</code> Ruby exception with the given …\nConstruct a new, <code>KeyError</code> Ruby exception with the given …\nConstruct a new, <code>LoadError</code> Ruby exception with the given …\nConstruct a new, <code>LocalJumpError</code> Ruby exception with the …\nConstruct a new, <code>NameError</code> Ruby exception with the given …\nConstruct a new, <code>NoMemoryError</code> Ruby exception with the …\nConstruct a new, <code>NoMethodError</code> Ruby exception with the …\nConstruct a new, <code>NotImplementedError</code> Ruby exception with …\nConstruct a new, <code>RangeError</code> Ruby exception with the given …\nConstruct a new, <code>RegexpError</code> Ruby exception with the given …\nConstruct a new, <code>RuntimeError</code> Ruby exception with the given\nConstruct a new, <code>ScriptError</code> Ruby exception with the given …\nConstruct a new, <code>SecurityError</code> Ruby exception with the …\nConstruct a new, <code>SignalException</code> Ruby exception with the …\nConstruct a new, <code>StandardError</code> Ruby exception with the …\nConstruct a new, <code>StopIteration</code> Ruby exception with the …\nConstruct a new, <code>SyntaxError</code> Ruby exception with the given …\nConstruct a new, <code>SystemCallError</code> Ruby exception with the …\nConstruct a new, <code>SystemExit</code> Ruby exception with the given …\nConstruct a new, <code>SystemStackError</code> Ruby exception with the …\nConstruct a new, <code>ThreadError</code> Ruby exception with the given …\nConstruct a new, <code>TypeError</code> Ruby exception with the given …\nConstruct a new, <code>UncaughtThrowError</code> Ruby exception with …\nConstruct a new, <code>ZeroDivisionError</code> Ruby exception with the …")